### ISSUE TITLE
Double Concentration Conditions

### ISSUE THREAD

**Issue by Gurrgel** [death-save/combat-utility-belt#662]
When using the Concentrator gadget from CUB, I get duplicates of the Concentration Effect whenever a token casts a concentration spell, removing one of them removes both of them, creating a lot of noise in the chat and extra visual noise on the token. I'm running both MidiQOL and CUB, but I don't have the Concentrator active from MidiQOL. From attempts to troubleshoot using Find the Culprit, there seems to be a conflict with the Better Rolls module.

Manage to reproduce the effect with only the following modules active, and disabling Better Rolls stops the problem from happening:

Better Rolls for 5e 1.7.2
libWrapper 1.12.4.0
Midi QOL 0.9.14
Midi SRD 0.1.03
socketlib 1.0.10
Times Up 0.8.31
Advanced Macros 1.14.1
Combat Utility Belt 1.8.2
Dynamic Active Effects SRD 6.0.01
Dynamic effects using Active Effects 0.9.07
Simple Calendar 1.3.75

Foundry v9 Build 249
Dnd 5e v1.5.7

**Comment by eclarke12** [comment]
@Gurrgel with just CUB and BR enabled I can't replicate this, can you try testing with just those two enabled and also provide your BR settings?

**Comment by LtDerek** [comment]
I've done alot of looking into this and the conflict is between CUB Concentrator and MIDIqol.

Even when MIDIqol concentrator is disable, there is a problem between the two modules that causes double concentration.

I was able to replicate it with only MIDIqol and CUB activated (and their requirements)

________________________

It's related to type of spell that is causing the concentration. Specifically, it seems to trigger when a spell is a saving throw, healing, ability check or utility WITH a damage formula.

It doesn't matter if the damage formula itself is blank or not.

Basically. If it doesn't have an attack roll, and DOES have a damage formula, the concentrator will trigger twice.

**Comment by eclarke12** [comment]
Thanks for the awesome investigation @LtDerek ! I will try to poke at this some more when I open up CUB next week.

**Comment by Cardagon** [comment]
I did some testing as I was getting this issue as well. I use midi-qol for its QOL and not roll automation so an easy way to fix this issue I found is to disable the setting "Enable roll automation support" under midi-qol. It still lets you use midi-qol for everything else, but I was not using any automated damage, targeting, anyways, and disabling this setting seemed to fix the duplicate concentration issue.

**Comment by Zealouzy** [comment]
This is still happening to me on ver. 1.9.2 CUB and 0.9.61 Midi-QoL.

CUB concentration automation enabled, Midi QoL disabled. Double Concentration happens on spells like those described above.
